packages/experiments/src/slots/examples/Slots.Recomposition.Example.tsx

```diff
@@ -0,0 +1,73 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { Button, IButtonComponent, IButtonProps, IButtonStyles, IButtonTokens } from '@uifabric/experiments';
+import { withSlots } from '@uifabric/foundation';
+// Temporary import file to experiment with next version of foundation.
+import { composed } from '@uifabric/foundation/lib/next/composed';
+import { buttonProperties, getNativeProps } from '@uifabric/utilities';
+import { Spinner, Stack, IStackProps } from 'office-ui-fabric-react';
+
+const stackProps: IStackProps = { tokens: { childrenGap: 16 }, padding: 8, maxWidth: 400 };
+
+const SpinnerButton: React.StatelessComponent<IButtonProps> = composed<IButtonProps, IButtonTokens, IButtonStyles>(Button, {
+  slots: { icon: Spinner }
+});
+
+const IconAtEndButtonView: IButtonComponent['view'] = (props, slots) => {
+  const { icon, content, children, disabled, onClick, allowDisabledFocus, ariaLabel, keytipProps, buttonRef, ...rest } = props;
+
+  const buttonProps = getNativeProps<React.HTMLAttributes<HTMLButtonElement>>(rest, buttonProperties);
+
+  const _onClick = (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement | HTMLDivElement>) => {
+    if (!disabled && onClick) {
+      onClick(ev);
+
+      if (ev.defaultPrevented) {
+        return;
+      }
+    }
+  };
+
+  return (
+    <slots.root
```

Comment:
I didn't think this was valid JSX; I thought that "slots" would need to be capitalized?

Reply:
Capitalization shouldn't matter as long as it's the right type.

Reply:
Well, this is just something that is being passed as a parameter into the view function. I can capitalize it if you want, the corresponding line is line 16.